va/va.go

```diff
@@ -524,3 +542,26 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 
 	return body, url.String(), nil
 }
+func alpnaddr(addr string) (arpa string, err error) {
```

Comment:
A few nits here:

1) Can you add an empty newline between this function's start and the previous function's end?
2) I think `alpnaddr` isn't the most descriptive name. I think it's computing an SNI value to send in a TLS-ALPN-01 validation request specifically for IP address validation, is that correct? If so then I think the new name needs to capture that its specific to ACME-IP.
3) Can you add a Godoc comment on this function? It's a significant bit of code and could use the added context/description.
4) Please don't use named return values (`arpa`, `err`). We avoid those in Boulder/Pebble as a matter of style. Thanks!